pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -67,6 +69,8 @@ public abstract class BaseBrokerRequestHandler implements BrokerRequestHandler {
   protected final BrokerRequestIdGenerator _requestIdGenerator;
   protected final long _brokerTimeoutMs;
   protected final QueryLogger _queryLogger;
+  @Nullable
+  protected final String _enableQueryNullHandling;
```

Comment:
(nit) Since it is already under the context of query handler, consider simplify it to `_enableNullHandling`